src/wp-includes/class-wp-block-type.php

```diff
@@ -327,6 +327,13 @@ public function set_props( $args ) {
 
 		$args['name'] = $this->name;
 
+		// Make sure core blocks register a stylesheet.
+		if ( 0 === strpos( $args['name'], 'core/' ) ) {
+			$block_name           = str_replace( 'core/', '', $args['name'] );
+			$args['style']        = ! empty( $args['style'] ) ? $args['style'] : "wp-block-$block_name";
+			$args['editor_style'] = ! empty( $args['editor_style'] ) ? $args['editor_style'] : "wp-block-$block_name-editor";
+		}
+
```

Comment:
Wasn't this added in `blocks.php` in https://core.trac.wordpress.org/changeset/51102

Should r51102 be reverted? Or this removed?

Reply:
You are correct, this part was already added in a previous patch. Removed it from this PR. Thank you for catching this! :+1: